src/FastExpressionCompiler.LightExpression/Expression.cs

```diff
@@ -57,14 +57,21 @@ public abstract class Expression
         internal static SysExpr[] ToExpressions(IReadOnlyList<Expression> exprs)
         {
             if (exprs.Count == 0)
+            {

```

Comment:
Why did added the brackets everywhere? Could revert this back?

Reply:
Oh dear! Sorry about that - must have been a VS clean up with CTRL+K, CTRL+D. I'll fix it.